patches/components-permissions-request_type.h.patch

```diff
@@ -1,12 +1,12 @@
 diff --git a/components/permissions/request_type.h b/components/permissions/request_type.h
-index d219ccbc3cbda2843ff24ade4d3c4cd04b29227b..fbc0466455326b7db406407247db17064016abe0 100644
+index d219ccbc3cbda2843ff24ade4d3c4cd04b29227b..4daf0d72e116af2a503829d85d0c17bbf06d24cc 100644
 --- a/components/permissions/request_type.h
 +++ b/components/permissions/request_type.h
-@@ -50,6 +50,7 @@ enum class RequestType {
+@@ -49,6 +49,7 @@ enum class RequestType {
+   kSecurityAttestation,
  #endif
    kStorageAccess,
-   kVrSession,
 +  BRAVE_REQUEST_TYPES
```

Comment:
why did you move this?

Reply:
Build error due to kMaxValue = kVrSession

Reply:
why not replace any value with a define and add our values in it? seems like the order is not important here.

Reply:
that might be a little tricky since we're also conditionally including each brave request type, but maybe still doable. Although, is it really necessary to conditionally include the request types? Doesn't seem like it should be.

Reply:
I haven't tried removing those buildflag guards as I was following existing WIDEVINE type.
I do prefer we dropping guards for just types like these, I'll make the change and double check if it still build fine on all platforms.

Reply:
Fixed in https://github.com/brave/brave-core/pull/9443/commits/f4de187d10c09900be539f83a54aa5efc1e340f6